samples/package.json

```diff
@@ -17,6 +17,7 @@
   ],
   "dependencies": {
     "@google-cloud/automl": "^2.2.0",
+    "@google-cloud/translate": "^6.0.2",
```

Comment:
huh, weird - don't think you need this, right?

Reply:
oh yea we do not need it, do not know what I was thinking